src/Cache_Command.php

```diff
@@ -151,6 +158,7 @@ public function trigger( $args, $assoc_args ) {
 		}
 
 		/** This filter is documented in wp-includes/class-wp-embed.php */
+		// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound
```

Comment:
Please add a comment stating why this is whitelisted.

```suggestion
		// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound -- Using WP Core hook.
```